vega_sim/scenario/constants.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum, auto
+
+
+class Network(Enum):
+    NULLCHAIN = "nullchain"
+    STAGNET1 = "vegawallet-stagnet1"
```

Comment:
Why isn't this just "stagnet1"?

Reply:
Ah yes so this is annoying, the config file for `stagnet1` is called `vegawallet-stagnet1.toml`, I would like to neaten the way VegaServiceNetwork reads config files in the future to make it a bit better and probably put this Enum somewhere else but for now it should be okay